docs/csharp/quick-starts/numbers-in-csharp-local.md

```diff
@@ -0,0 +1,373 @@
+---
+title: Quick Start - Numbers in C# - C# Guide
+description: Learn C# by exploring numeric types, their properties and methods.
+author: billwagner
+ms.author: wiwagn
+ms.date: 10/31/2017
+ms.topic: article
+ms.prod: .net
+ms.technology: devlang-csharp
+ms.devlang: csharp
+ms.custom: mvc
+---
+
+# Numbers in C# quick start #
+
+This quick start teaches you about the number types in C# interactively. You'll write small amountes of code, then you'll compile and run that code. It contains a series of lessons that explore numbers and math operations in C#. These lessons teach you the fundamentals of the C# language.
+
+This quick start expects you to have a machine you can use for development. The .NET topic [Get Started in 10 minutes](https://www.microsoft.com/net/core) has instructions for setting up your local development environment on Mac, PC or Linux.
+
+## Explore integer math
+
+Create a directory named **numbers-quickstart**. Make that the current directory and run `dotnet new console -n NumbersInCSharp -o .`.
+
+Open **Program.cs** in your favorite editor, and replace the line `Console.Writeline("Hello World!");` with the following:
+
+```csharp
+int a = 18;
+int b = 6;
+int c = a + b;
+Console.WriteLine(c);
+```
+
+Run this code by typing `dotnet run` in your command window. 
+
+You've just seen one of the fundamental math operations with integers. The `int` type represents an **integer**, a positive or negative whole number. You use the `+` symbol for addition. Other common mathematical operations for integers include:
+
+- `-` for subtraction
+- `*` for multiplication
+- `/` for division
+
+Start by exploring those different operations. Add these lines after the line that writes the value of `c`:
+
+```csharp
+c = a - b;
+Console.WriteLine(c);
+c = a * b;
+Console.WriteLine(c);
+c = a / b;
+Console.WriteLine(c);
+```
+
+Run this code by typing `dotnet run` in your command window. 
+    
+You can also experiment by performing multiple mathematics operations in the same line, if you'd like.
+
+> [!TIP]
+> As you explore C# (or any programming language), you'll
+> make mistakes when you write code. The **compiler** will
+> find those errors and report them to you. When the output
+> contains error messages, look closely at the example code,
+> and the code in the interactive window to see what to fix.
+> That exercise will help you learn the structure of C# code.     
+
+You've finished the first step. Before you start the next section, let's move the current code into a separate method. That makes it easier to start working with a new example. Rename your `Main` method to `WorkWithIntegers` and write a new `Main` method that calls `WorkWithIntegers`. When you have finished, your code should look like this:
```

Comment:
This is `WorkWithIntegers`, but the code shows `WorkingWithIntegers`.

Reply:
I went with the code as correct. :)